README.md

```diff
@@ -157,76 +229,27 @@ Rosetta requires a few Celo Core Contracts
 
 * The list of required contracts is defined on `scripts/gen-contracts.go` file
 * Generation requires acces to `celo-blockchain` & `celo-monorepo`.
-* Generation assumes both projects are **already properly built**
+* Generation assumes both projects are **already properly built** (see above under instructions for running `rosetta` from source for more details on how to do this)
 * To run generator do `make gen-contracts`
 
-## How to build Docker Image
-
-Commands:
-  * `make docker-build`
-  * `make docker-publish`
-
 ## How to run rosetta-cli-checks
+
 Install the [`rosetta-cli`](https://github.com/coinbase/rosetta-cli) according to the instructions. Current testing has been done with `v0.5.16`.
-- Run the Rosetta service in the background for the respective network (currently only alfajores)
-- Run the CLI checks as follows:
-```
+
+* Run the Rosetta service in the background for the respective network (currently only alfajores)
+* Run the CLI checks as follows:
+
+```sh
 # alfajores; specify construction or data
 rosetta-cli check:construction --configuration-file PATH/TO/rosetta/rosetta-cli-conf/testnet/cli-config.json
```

Comment:
I'm actually not sure if I can run checks becaus eI just get a bunch of these
```sh
2021/01/28 14:48:08 waiting for implementation to reach tip before testing...
[MEMORY] Heap: 1103.278084MB Stack: 0.687500MB System: 1260.756973MB GCs: 5
[STATS] Transactions Confirmed: 0 (Created: 0, In Progress: 0, Stale: 0, Failed: 0) Addresses Created: 0
2021/01/28 14:48:18 waiting for implementation to reach tip before testing...
```

How long does it normally take before it reaches the tip? Should I leave it running for a while? If so, I'd add something about that

Reply:
Yes you're all good! It depends on whether you're starting with a fresh node or one with data, etc. Will add a note about that, but yes if this is the first time it'll take quite a while to reach the tip (syncing a full archive node). Data Checks themselves will take a long time since it needs to go through the entire chain + reconcile balances.

Reply:
✅ added a note about this